.github/workflows/spack.yml

```diff
@@ -25,18 +27,49 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ~/.spack-cache
-          key: spack-cache-${{ matrix.os }}-${{ github.run_id }}
-          restore-keys: spack-cache-${{ matrix.os }}-
-
-      - name: Build Arbor's Spack package against the develop branch
-        run: arbor/scripts/build_spack_package.sh arbor develop
-
-      # build_spack_package.sh only builds, does not install, therefore can't run Python code.
-      # - name: Run Python examples
-      #   run: |
-      #     cd arbor
-      #     scripts/run_python_examples.sh
-      # - name: Test executables
-      #   run: |
-      #     cd arbor
-      #     scripts/test_executables.sh
+          key: spack-cache-${{ matrix.os }}-${{ matrix.spack-version }}-${{ github.run_id }}
+          restore-keys: spack-cache-${{ matrix.os }}-${{ matrix.spack-version }}-
+      - name: Get Spack (develop)
+        if: ${{ matrix.spack-version == 'develop' }}
+        run: |
+          git clone -c feature.manyFiles=true --depth 1 https://github.com/spack/spack.git
+      - name: Get Spack (latest_release)
+        if: ${{ matrix.spack-version == 'latest_release' }}
+        run: |
+          wget "$(curl -sH "Accept: application/vnd.github.v3+json" https://api.github.com/repos/spack/spack/releases/latest | grep browser_download_url |  cut -d '"' -f 4)"
```

Comment:
wuhu.

Reply:
1. I understand, BUT:
2. We were already doing this (see `build_spack_package.sh`)
3. It is done through a GitHub API, not some random website. It's the recommended way to get release artifacts.